pandas/tests/scalar/test_na_scalar.py

```diff
@@ -69,6 +72,49 @@ def test_comparison_ops():
         assert (other <= NA) is NA
 
 
+@pytest.mark.parametrize(
```

Comment:
There are fixtures defined for zero / one values in pandas/tests/indexing/conftest.py - can these be combined with that?

Reply:
That seems to include arrays. We just want a scalar here I think.